src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/controllers/InitialConsiderationController.java

```diff
@@ -0,0 +1,52 @@
+package uk.gov.hmcts.ethos.replacement.docmosis.controllers;
+
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.media.Content;
+import io.swagger.v3.oas.annotations.media.Schema;
+import io.swagger.v3.oas.annotations.responses.ApiResponse;
+import io.swagger.v3.oas.annotations.responses.ApiResponses;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestHeader;
+import org.springframework.web.bind.annotation.RestController;
+import uk.gov.hmcts.et.common.model.ccd.CCDCallbackResponse;
+import uk.gov.hmcts.et.common.model.ccd.CCDRequest;
+import uk.gov.hmcts.ethos.replacement.docmosis.service.InitialConsiderationService;
+import uk.gov.hmcts.ethos.replacement.docmosis.service.VerifyTokenService;
+
+import static org.springframework.http.HttpStatus.FORBIDDEN;
+import static org.springframework.http.MediaType.APPLICATION_JSON_VALUE;
+
+@Slf4j
+@RequiredArgsConstructor
+@RestController
+public class InitialConsiderationController {
+    private final VerifyTokenService verifyTokenService;
+    private final InitialConsiderationService initialConsiderationService;
+
+    private static final String INVALID_TOKEN = "Invalid Token {}";
+
+    @PostMapping(value = "/completeInitialConsideration", consumes = APPLICATION_JSON_VALUE)
+    @Operation(summary = "completes the Initial Consideration flow")
+    @ApiResponses(value = {@ApiResponse(responseCode = "200", description = "Accessed successfully", content = {
+        @Content(mediaType = "application/json", schema = @Schema(implementation = CCDCallbackResponse.class))}),
+        @ApiResponse(responseCode = "400", description = "Bad Request"),
+        @ApiResponse(responseCode = "500", description = "Internal Server Error")})
+    public ResponseEntity<CCDCallbackResponse> completeInitialConsideration(@RequestBody CCDRequest ccdRequest,
+                                                                            @RequestHeader(value = "Authorization")
+                                                                                String userToken) {
+        log.info("Initial consideration complete requested for case reference ---> " + ccdRequest.getCaseDetails().getCaseId());
```

Comment:
Better to use the newer way of logging:
log.info("Initial consideration complete requested for case reference ---> ", ccdRequest.getCaseDetails().getCaseId());

Reply:
fixed